### PR TITLE
Audit split ui

### PR DESCRIPTION
**Pull request by HEskandari**

### CODE REVIEW

File: src/ServiceControl.Config/Commands/UpgradeServiceControlInstanceCommand.cs [hunk 1]
```diff
@@ -151,7 +151,7 @@ public override async Task ExecuteAsync(InstanceDetailsViewModel model)
                 }
             }
 
-            if (!instance.AppConfig.AppSettingExists(SettingsList.DatabaseMaintenancePort.Name))
+            if (!instance.AppConfig.AppSettingExists(AuditInstanceSettingsList.DatabaseMaintenancePort.Name))
```
Comment (mikeminutillo) [inline src/ServiceControl.Config/Commands/UpgradeServiceControlInstanceCommand.cs]: Should this be AuditInstanceSettingsList? All of the other options in the file are ServiceControlSettings.

File: src/ServiceControl.Config/UI/InstanceEdit/ServiceControlAuditEditAttachment.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            instance.Description = viewModel.ServiceControlAudit.Description;
+            instance.HostName = viewModel.ServiceControlAudit.HostName;
+            instance.Port = Convert.ToInt32(viewModel.ServiceControlAudit.PortNumber);
+            instance.DatabaseMaintenancePort = !string.IsNullOrWhiteSpace(viewModel.ServiceControlAudit.DatabaseMaintenancePortNumber) ? Convert.ToInt32(viewModel.ServiceControlAudit.DatabaseMaintenancePortNumber) : (int?)null;
+            instance.AuditLogQueue = viewModel.ServiceControlAudit.AuditForwardingQueueName;
+            instance.AuditQueue = viewModel.ServiceControlAudit.AuditQueueName;
+            instance.ForwardAuditMessages = viewModel.ServiceControlAudit.AuditForwarding.Value;
+            instance.AuditRetentionPeriod = viewModel.ServiceControlAudit.AuditRetentionPeriod;
+            instance.TransportPackage = viewModel.SelectedTransport;
+            instance.ConnectionString = viewModel.ConnectionString;
```
Comment (mikeminutillo) [inline src/ServiceControl.Config/UI/InstanceEdit/ServiceControlAuditEditAttachment.cs]: Can we not edit the audit instance ServiceControlAddress?

File: src/ServiceControl.Config/ServiceControl.Config.csproj [hunk 3]
```diff
@@ -45,6 +45,7 @@
 
   <ItemGroup>
     <None Remove="App.ico" />
+    <None Remove="UI\InstanceEdit\ServiceControlAuditEditView.xaml" />
```
[the remark this answers is not in the document]
Comment (HEskandari) [reply src/ServiceControl.Config/ServiceControl.Config.csproj]: Think it was added by Resharper. I'll have a look.

File: src/ServiceControl.Config/UI/AdvancedOptions/ServiceControlAdvancedViewModel.cs [hunk 4]
```diff
@@ -46,7 +45,7 @@ public ServiceControlAdvancedViewModel(BaseService instance, IEventAggregator ev
         public ICommand StartServiceInMaintenanceModeCommand { get; set; }
         public ICommand StopMaintenanceModeCommand { get; set; }
 
-        public bool MaintenanceModeSupported => ServiceControlInstance.Version >= SettingsList.MaintenanceMode.SupportedFrom;
```
[the remark this answers is not in the document]
Comment (HEskandari) [reply src/ServiceControl.Config/UI/AdvancedOptions/ServiceControlAdvancedViewModel.cs]: Brought this back.

File: src/ServiceControl.Config/ServiceControl.Config.csproj (hunk 3 above)
[the remark this answers is not in the document]
Comment (HEskandari) [reply src/ServiceControl.Config/ServiceControl.Config.csproj]: Removed this from the project file.

**Pull request merged**